src/lib/utils.ts

```diff
@@ -44,15 +44,19 @@ export const formatTimestamp = (timestamp: number) =>
   })} UTC`;
 
 export const getAmountWithCurrencySymbol = (
-  amount: bigint,
-  currency: string
+  amount: bigint | null | undefined,
+  currency: string | null | undefined
 ) => {
+  if (!amount) return "0";
```

Comment:
Can we make it show unknown or undefined and log an error? The payment amount should never be null.